src/frontend/screens/Settings/ProjectSettings/index.tsx

```diff
@@ -1,112 +1,209 @@
+import {useProjectSettings} from '../../../hooks/server/projects';
+import {useNavigationFromRoot} from '../../../hooks/useNavigationWithTypes';
 
 const m = defineMessages({
   title: {
-    id: 'Screens.Settings.ProjectSettings.title',
+    id: 'Screens.ProjectSettings.title',
     defaultMessage: 'Project Settings',
   },
-  deviceName: {
-    id: 'Screens.Settings.ProjectSettings.deviceName',
-    defaultMessage: 'Device Name',
+  soloDescription: {
+    id: 'Screens.ProjectSettings.soloDescription',
+    defaultMessage: 'You’re mapping on your own.',
   },
-  yourTeam: {
-    id: 'Screens.Settings.ProjectSettings.yourTeam',
-    defaultMessage: 'Your Team',
+  coordinator: {
+    id: 'Screens.ProjectSettings.coordinator',
+    defaultMessage: 'This device is a coordinator on this project.',
   },
-  mediaSyncSettings: {
-    id: 'Screens.Settings.ProjectSettings.mediaSyncSettings',
-    defaultMessage: 'Sync Settings',
+  participant: {
+    id: 'Screens.ProjectSettings.participant',
+    defaultMessage: 'This device is a participant on this project.',
   },
-  config: {
-    id: 'screens.Settings.config',
-    defaultMessage: 'Configuration',
-    description: 'Primary text for project config settings',
+  invite: {
+    id: 'Screens.ProjectSettings.invite',
+    defaultMessage: 'Invite Collaborators',
   },
-  RemoteArchive: {
-    id: 'Screens.Settings.ProjectSettings.RemoteArchive',
-    defaultMessage: 'Remote Archive',
+  projectCollaborators: {
+    id: 'Screens.ProjectSettings.projectCollaborators',
+    defaultMessage: 'Project Collaborators',
   },
-  remoteArchiveOff: {
-    id: 'Screens.Settings.ProjectSettings.remoteArchiveOff',
-    defaultMessage: 'Remote Archive is OFF',
+  viewTeam: {
+    id: 'Screens.ProjectSettings.viewTeam',
+    defaultMessage: 'View Team',
   },
-  remoteArchiveOn: {
-    id: 'Screens.Settings.ProjectSettings.remoteArchiveOn',
-    defaultMessage: 'Remote Archive is ON',
+  configTitle: {
+    id: 'Screens.ProjectSettings.configTitle',
+    defaultMessage: 'Project Categories',
+  },
+  updateCategories: {
+    id: 'Screens.ProjectSettings.updateCategories',
+    defaultMessage: 'Update Set',
+  },
+  editInfo: {
+    id: 'Screens.ProjectSettings.editInfo',
+    defaultMessage: 'Edit Info',
   },
 });
 
-export const ProjectSettings: NativeNavigationComponent<'ProjectSettings'> = ({
-  navigation,
+export const ProjectSettings = () => {
+  const {projectId} = useActiveProject();
+  const projectInfo = useProjectRoleAndDetails(projectId);
+
+  const displayTitle =
+    projectInfo.role === 'solo'
+      ? projectInfo.projectHeader
+      : projectInfo.projectName;
+
+  return (
+    <ScrollView contentContainerStyle={styles.container}>
+      <ProjectInfoCard role={projectInfo.role} displayTitle={displayTitle} />
+      {projectInfo.role !== 'solo' && (
+        <CollaboratorsCard role={projectInfo.role} />
+      )}
+      {projectInfo.role !== 'participant' && <ConfigCard />}
```

Comment:
I think these are unnecessarily prop forwarding.

To follow our guidelines, i don't think we should create these "cards", and instead just use 3 `SettingsCardRow` and do all the logic at the top level of this component